File: Labs/Lab01-C/lab01.c
#include <stdio.h>


    void maiusculo(char palavra[], int cont){
    
        int contar = 0;

        for(int i = 0; i < cont; i++){

            if(palavra[i] >= 65 && palavra[i] <= 90){

                contar++;

            }
            
        }

            printf("Quantidade de Letras MAIUSCULAS: %d",contar);

            printf("\nDigite a palavra: ");
            scanf("%s", palavra);

    }


    int main(){

        int cont = 0;

        printf("Digite a palavra: ");
        char palavra[100];
        scanf("%s", palavra);
        
        while(palavra[cont] != '\0'){
            cont++;
        }

        
        while (palavra[0] != 'F' && palavra[1] != 'I' && palavra[2] != 'M' && palavra[3] != '\0'){ 
            
           
            maiusculo(palavra,cont);
             
            
        }

         return 0;
        
    }
    

File: Labs/Lab01-Java/lab01.java
import java.util.Scanner;

public class lab01 {

    public static void maiusculo(String palavra){
    
       
        int cont = 0;

        for(int i = 0; i < palavra.length(); i++){

            if(palavra.charAt(i) >= 65 && palavra.charAt(i) <= 90 ){

                cont++;

            }
            
        }

            System.out.println("Quantidade de Letras MAIUSCULAS: " + cont);

    }


    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite a palavra");
        String palavra= scanner.nextLine();

        
        
        while (!palavra.equals("FIM")){ //Equals compara o conteudo de dois objetos

            maiusculo(palavra);

            System.out.println("Digite a palavra");
            palavra= scanner.nextLine();
        }

         scanner.close();
        
    }
    
}
